contents/docs/libraries/react-native/index.mdx

```diff
@@ -15,22 +15,104 @@ features:
   groupAnalytics: true
 ---
 
-Purely built for React Native, this library allows you to integrate PostHog with your React Native project. For React Native projects built with Expo, there are no mobile native dependencies outside of supported Expo packages.
```

Comment:
It's ok if we simplify this sentence but I'd not remove it altogether since it says that it's Expo compatible and there are no native deps which is important for some people.